crates/standalone/src/lib.rs

```diff
@@ -163,6 +163,8 @@ fn get_key_path(env: &str) -> Option<PathBuf> {
 #[async_trait]
 impl spacetimedb_client_api::NodeDelegate for StandaloneEnv {
     fn gather_metrics(&self) -> Vec<prometheus::proto::MetricFamily> {
+        // Note, we update certain metrics such as disk usage on demand.
+        self.db_inst_ctx_controller.update_metrics();
```

Comment:
I'm guessing this thing is called slowly enough that `stat`ing a bunch of stuff is not going to be a big deal, but I'd like to make sure that's an ok assumption.